### PR TITLE
[WIP] [12.0] [IMP] pos_picking_delayed: Configuring pricelist on test

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26576

@chienandalu this is the improvement for avoid the errors on tests.

### CODE REVIEW

File: pos_picking_delayed/tests/test_module.py [hunk 1]
```diff
@@ -12,10 +12,25 @@ def setUp(self):
...
-        self.pricelist = self.env.ref('product.list0')
+        self.pricelist = self.env['product.pricelist'].create({
+            'name': 'Test pricelist',
+            'item_ids': [(0, 0, {
+                'applied_on': '3_global',
+                'compute_price': 'formula',
+                'base': 'list_price',
+            })]
+        })
+        self.env['res.partner'].create({
```
Comment (pedrobaeza) [inline pos_picking_delayed/tests/test_module.py]: Why creating this partner if you are not going to use it?

Comment (CarlosRoca13) [reply pos_picking_delayed/tests/test_module.py]: I thought that was important, I removed yet

**Review by pedrobaeza — APPROVED** [review]

**Review by legalsylvain — APPROVED** [review]
Don't understand. travis seems green on V12 branch : https://travis-ci.org/github/OCA/pos/branches

otherwise, LGTM. thanks !

**Comment by chienandalu** [comment]
> Don't understand. travis seems green on V12 branch : https://travis-ci.org/github/OCA/pos/branches
> 
> otherwise, LGTM. thanks !

Yes, @legalsylvain but depending on your environment default localization it can vary.

**Comment by pedrobaeza** [comment]
@legalsylvain this happens on integration CIs where other modules from other repositories/odoo core are installed among this one. Using demo/core data has this risk, so tests must tend to be self-contained.

**Comment by legalsylvain** [comment]
ok thanks for the detail.

fastracking trivial patch.

/ocabot merge patch

**Comment by CarlosRoca13** [comment]
@pedrobaeza Please cancel the merge i'm keeping improving it

**Comment by pedrobaeza** [comment]
Done. Next time, please label it as WIP or draft PR if you are not sure of the changes.

**Review by chienandalu — APPROVED** [review]

**Pull request merged**